packages/block-editor/src/components/iframe/content.scss

```diff
@@ -1,5 +1,12 @@
 .block-editor-iframe__body {
 	position: relative;
+
+	// Float an outline on the top of the iframe to show focus on the canvas when pressing
+	// escape while in select mode.
+	&:focus::after {
+		@include selectedOutline();
+		position: fixed;
+	}
```

Comment:
Is this potentially too broad? I notice that in the post editor, if I have multiple blocks selected in the canvas, then the iframe outline is also rendered, which looks unexpected:

<img width="1274" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/1a7c724b-09ab-41e8-a83b-913574f1205c">